pkg/summary/summarizers.go

```diff
@@ -74,14 +81,26 @@ var (
 		"ReplicaFailure":      true,
 		"Stalled":             true,
 	}
```

Comment:
Should this list here be reworked to use the same logic? Doesn't feel right to have a separate legacy list of node status error mappings, now that we have a proper way to map resource status conditions to errors.